epoi/inject/inject.py

```diff
@@ -50,15 +50,36 @@ def __init__(self, policies=None):
         for policy_cls in policies if policies is not None else get_activate_policies():
             self.policies.append(policy_cls())
         self.record = {}
+        self.load_state_dict_backup = None
 
     def __enter__(self):
         for policy in self.policies:
             policy.hook(self)
+        self.inject_load_state_dict()
+
+    def inject_load_state_dict(self):
+        import transformers.modeling_utils
+        self.load_state_dict_backup = transformers.modeling_utils.load_state_dict
+
+        def wrap_load_state_dict(orig_load_func, policies):
+            def wrap_load_func(*args, **kwargs):
+                state_dict = orig_load_func(*args, **kwargs)
+                for policy in policies:
+                    state_dict = policy.load_state_dict_post_hook(state_dict)
```

Comment:
`load_state_dict_post_hook` from different models' policies may conflict, say, if users pass the injection policies of bert and bloom into the context and if their parameter names conflict.

I didn't check such potential conflict, because HF gives detailed warnings if there is any mismatch between the loaded state_dict and the model parameters.

Reply:
Reasonable. Could you make this comment in code for better checking in the future?